features/exploredash/src/main/res/layout/fragment_purchase_gift_card.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/title_bar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        app:layout_constraintTop_toTopOf="parent"
+        app:navigationIcon="@drawable/ic_arrow_back_primary" />
+
+    <org.dash.wallet.common.ui.PaymentHeaderView
+        android:id="@+id/payment_header_view"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/title_bar" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/enter_amount_guideline"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        app:layout_constraintGuide_percent="0.18" />
```

Comment:
I think 18% isn't enough. On tall screens it doesn't look good - it's too close to the header. The amount should be somewhere in the center between the header and the keyboard.

<img src="https://user-images.githubusercontent.com/7733989/212071225-35e9fd79-627e-4817-8454-6d980a6c135a.jpeg"  width="300" />

Reply:
Fixed